compiler/rustc_target/src/spec/targets/x86_64_pc_windows_msvc.rs

```diff
@@ -9,8 +9,8 @@ pub fn target() -> Target {
     Target {
         llvm_target: "x86_64-pc-windows-msvc".into(),
         pointer_width: 64,
-        data_layout: "e-m:w-p270:32:32-p271:32:32-p272:64:64-i64:64-f80:128-n8:16:32:64-S128"
-            .into(),
+        data_layout:
```

Comment:
I believe win x86 targets are missing here https://github.com/bridiver/rust/commit/defeeb914e043281953849740e5507ff60f9c2ee#diff-82540e5fa4a277a476dc2b2e408d3633344032cb745bf7d2013fd27e6bc62247

Reply:
They should be there now https://github.com/rust-lang/rust/blob/2f13a43ac19434192183b455ef2b2e98ae7a68e0/compiler/rustc_target/src/spec/targets/i686_pc_windows_gnu.rs#L22